classes/phing/tasks/ext/SymlinkTask.php

```diff
@@ -236,7 +236,36 @@ protected function getMap()
             }
 
             if ($this->isRelative()) {
-                $fromDir = $fs->getDir($this->getProject())->getPath();
+                $endPath = $fs->getDir($this->getProject())->getAbsolutePath();
+                $startPath = $this->getLink();
```

Comment:
You can get the absolute path for `$this->getLink()` in combination with `PhingFile` like:
```
$startPath = (new PhingFile($this->getLink()))->getAbsolutePath();
```